cmd/pool/order.go

```diff
@@ -116,6 +116,20 @@ var sharedFlags = []cli.Flag{
 			channelTypePeerDependent,
 			channelTypeScriptEnforced),
 	},
+	cli.StringSliceFlag{
+		Name: "allowed_node_id",
+		Usage: "the list of nodes this order is allowed to match " +
+			"with; if empty, the order will be able to match " +
+			"with any node unless not_allowed_node_id is set. " +
+			"Can be specified multiple times",
+	},
+	cli.StringSliceFlag{
+		Name: "not_allowed_node_id",
```

Comment:
Not sure about this one. I could change it only in the CLI, but it could leak from some `rpc_server` errors and potentially confuse the user.  For example, parsing the order could return `invalid not_allowed_node_id`. 

Is it because for native speakers `prohibited_node_id` sounds better than `not_allowed_node_id`? If so I can change it everywhere, just not sure if it is just a matter of taste.

Reply:
Yeah, I'm not sure either given I'm not native but just thought it's simpler as "prohibited" is just one word. Regardless I'm fine with the current choice too.

Reply:
Prohibited sounds good to me. Or just disallowed. "not allowed" does sound a bit bumpy.

Reply:
But this probably also requires quite a big refactor/renaming so we can also just leave it the way it is.